backend/src/nodes/nodes/image/image_file_iterator.py

```diff
@@ -84,23 +84,7 @@ async def run(self, directory: str, context: IteratorContext) -> None:
 
         supported_filetypes = get_available_image_formats()
 
-        def walk_error_handler(exception_instance):
-            logger.warning(
-                f"Exception occurred during walk: {exception_instance} Continuing..."
-            )
-
-        just_image_files: List[str] = []
-        for root, dirs, files in os.walk(
-            directory, topdown=True, onerror=walk_error_handler
-        ):
-            await context.progress.suspend()
-
-            dirs.sort(key=alphanumeric_sort)
-            for name in sorted(files, key=alphanumeric_sort):
-                filepath = os.path.join(root, name)
-                _base, ext = os.path.splitext(filepath)
-                if ext.lower() in supported_filetypes:
-                    just_image_files.append(filepath)
+        just_image_files: List[str] = walk_sorted(directory, supported_filetypes)
```

Comment:
We should probably also use this for the Pytorch model iterator as well. It's kinda weird that it's different in that regard.

Reply:
True, tbh I forgot that one existed